src/connectionTool/constants/FolderConstant.java

```diff
@@ -0,0 +1,9 @@
+package connectionTool.constants;
+
+public class FolderConstant {
+
+    public static final String MAIN_DRIVER_PATH_WINDOWS = "C:\\Program Files\\dynatrace\\remotepluginmodule\\agent\\res\\java\\libs";
+    public static final String MAIN_DRIVER_PATH_LINUX = "\\var\\lib\\dynatrace\\remotepluginmodule\\agent\\res\\java\\libs\\";
```

Comment:
where do you have it from?
it should be: /opt/dynatrace/remotepluginmodule/agent/res/java/libs/